tools/evict-shared-cache/evict-shared-cache.cpp

```diff
@@ -108,6 +96,10 @@ struct CommandParser {
         }
         iter = end + 1;
       }
+
+      if (count < 4096) {
```

Comment:
Something like this is required or we will not process any messages until just before exiting

Reply:
Yeah because I misunderstood the read semantics I agree now.